src/styles/createTypography.ts

```diff
@@ -76,7 +81,8 @@ export interface TypographyOptions
     MUITypographyOptions {}
 
 const baseTypography: TypographyOptions = {
-  fontFamily: fontFamilies.sansSerif,
```

Comment:
Wait, don't we still need to override MUI's `fontFamily`. I believe removing this will include MUIs font family. I am doing this AFK so may be wrong.

Reply:
This might be a good reason to avoid the breaking change and keep primary as just `fontFamily` without the `primary` suffix.

Reply:
Ok, I think we're in a good spot now. We need to override MUI `fontFamily` so `fontFamily` will serve as Chroma's primary font. Thanks @HaleyWardo for all your help! 🙇🏻